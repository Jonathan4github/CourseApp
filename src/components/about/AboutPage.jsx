import React, { Fragment } from 'react';


const AboutPage = () => (
  <div className="jumbotron">
    <h1 className="display-4">About Us</h1>
    <p>The application uses react, redux and other libraries</p>
  </div>
);

export default AboutPage;
