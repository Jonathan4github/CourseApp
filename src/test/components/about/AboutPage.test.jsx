/* eslint-disable */
import AboutPage from '../../../components/about/AboutPage';


describe('<AboutPage /> component test', () => {
  it('renders page header for profile component without crashing', () => {
    const shallowWrapper = shallow(<AboutPage />);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
