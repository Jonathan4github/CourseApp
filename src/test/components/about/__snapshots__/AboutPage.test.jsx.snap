// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> component test renders page header for profile component without crashing 1`] = `
<div
  className="jumbotron"
>
  <h1
    className="display-4"
  >
    About Us
  </h1>
  <p>
    The application uses react, redux and other libraries
  </p>
</div>
`;
